x/ibc/20-transfer/keeper/relay.go

```diff
@@ -190,7 +186,8 @@ func (k Keeper) createOutgoingPacket(
 		packetDataBz,
 	)
 
-	// generate the capability key
+	// TODO: Remove this, capability keys are never generated when sending packets. Not sure why this is here.
```

Comment:
we should address this `TODO`

Reply:
We should indeed! I don't understand this line of code - what was it for?

Reply:
I didn't know where to create the capability key lol